### PR TITLE
Fix logout button.

### LINKED ISSUE #61: Logout button doesn't redirect without setting redirect_to.
The logout button should redirect to the app home page by default. Works when you set `redirect_to` to something else than the app url.

### LINKED ISSUE #62: Logout from on-prem with self signed certificates.
Logout fails when on-prem is using a self signed ssl certificate because it can't verify.

### PR DESCRIPTION
**Pull request by T4rk1n**
Two fixes for the logout button.

- When on-prem has a self signed certificate, the request to invalidate the token must not apply ssl verification. Fix #62.
- Append time query string to the default redirect of the app url. Fix #61.
- Use `requests_pathname_prefix` as path when clearing cookies.

- [x] test on DDS 2.5.0

### CODE REVIEW

**Comment by T4rk1n** [comment]
>Is there an issue related to this PR? (It's strongly preferable for all PRs to have an issue logged first, and in this particular case it would be great to see some examples of the problem you're trying to solve.)

I stumbled upon this bug when I added the logout button to an app in an on-prem instance that have a self signed certificate, I fixed quick and tested it and it resolved the bug so I thought I just make the PR.

Aside from the use in dash-auth, I don't see it used anywhere else. I had it removed before getting this bug and it was still working.
 
I'll try  `DASH_STREAMBED_DIRECT_IP` instead thanks.

**Comment by T4rk1n** [comment]
@scjody 
I changed to use the `DASH_STREAMBED_DIRECT_IP` and fixed another bug I found while testing #61. Please review.

File: dash_auth/plotly_auth.py [hunk 1]
```diff
@@ -155,6 +159,7 @@ def logout(self):
         invalidation_resp = requests.post(
             '{}{}'.format(api_requests.config('plotly_domain'),
                           '/o/revoke_token/'),
+            verify='DASH_STREAMBED_DIRECT_IP' not in os.environ,
```
Comment (scjody) [inline dash_auth/plotly_auth.py]: It doesn't appear that you're using `DASH_STREAMBED_DIRECT_IP` to connect to streambed here. As I said in https://github.com/plotly/dash-auth/pull/56#pullrequestreview-148690829, this is the _only_ case in which it's safe to make unverified requests. I'm happy to meet and discuss this if it's unclear...

**Comment by T4rk1n** [comment]
> I don't understand why you've changed _app_url to support being a list/tuple, and why the first element of that list/tuple is special. (Also in that case wouldn't _app_urls be a better name?)

I did not change that, the _app_url is in oauth and can be a string or a list of urls in the examples we have. In the case it's a list of urls I take the first one as default for the logout redirect.

**Comment by T4rk1n** [comment]
@scjody 
I changed the use the ip and tested on DDS 2.5.0, the logout work.

**Pull request merged**